python/helios/scene.py

```diff
@@ -139,6 +139,75 @@ def from_tiffs(cls, tiff_files: MultiAssetPath):
         Supports '**' for matching multiple directories.
         """
         return [ScenePart.from_tiff(tiff) for tiff in tiff_files]
+    
+
+    @classmethod
+    @validate_call
+    def from_xyz(
+        cls,
+        xyz_file: AssetPath, 
+        voxel_size: PositiveFloat,
+        separator: Optional[str] = None,
+        max_color_value: Optional[NonNegativeFloat] = 0., 
+        default_normal: Optional[NDArray[Shape["3"], np.float64]] = np.array([np.finfo(np.float64).max] * 3, dtype=np.float64),
+        sparse: Optional[bool] = False,
+        estimate_normals: Optional[bool] = False,
+        normals_file_columns: Optional[list[NonNegativeInt]] = [3, 4, 5],
+        rgb_file_columns: Optional[list[NonNegativeInt]] = [6, 7, 8],
+        snap_neighbor_normal: Optional[bool] = False,
```

Comment:
I think some of these annotations are off: `Optional[T]` is equivalent to `T|None`, but many of these do not actually accept `None`. I think only `spearator` really does, all the other ones have non-None defaults.